app/celery/provider_tasks.py

```diff
@@ -92,11 +92,15 @@ def send_sms_to_provider(self, service_id, notification_id):
         except SmsClientException as e:
             try:
                 current_app.logger.error(
-                    "SMS notification {} failed".format(notification_id)
+                    "RETRY: SMS notification {} failed".format(notification_id)
```

Comment:
this'll log with 'RETRY' even if it's the first time running the task - is that intended behaviour? if so why bother adding 'RETRY'?